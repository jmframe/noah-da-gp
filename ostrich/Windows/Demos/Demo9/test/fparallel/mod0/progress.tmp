          10  2.531724164286073E-018  1.300363244624236E-003
