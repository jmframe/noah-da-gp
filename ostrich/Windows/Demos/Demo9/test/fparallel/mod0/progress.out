 solve
     1      0.232E+04      0.130E-02      0.000E+00
     2      0.190E-05      0.127E-02      0.000E+00
     3      0.714E-07      0.129E-02      0.000E+00
     4      0.209E-08      0.129E-02      0.000E+00
     5      0.714E-10      0.128E-02      0.000E+00
     6      0.246E-11      0.127E-02      0.000E+00
     7      0.846E-13      0.127E-02      0.000E+00
     8      0.291E-14      0.131E-02      0.000E+00
     9      0.990E-16      0.132E-02      0.000E+00
    10      0.253E-17      0.130E-02      0.000E+00
 done
 finished
